ingest/ingest_pipeline.py

```diff
@@ -7,11 +7,18 @@
 
 PREREQUISITES
 You must have Google Cloud Firestore installed, authenticated, and
-configured. Must have Python 3.6 or higher.
+configured. Must have python 3.6 or higher. Indexing must be turned off for
+all collections.
 
 EXAMPLES
 # Takes expression file and stores it into Firestore
 
+# Ingest cluster file
+python ingest_pipeline.py ingest_cluster --cluster-file ../tests/data/AB_toy_data_portal.cluster.txt
+
+# Ingest Cell Metadata file
+python ingest_pipeline.py ingest_cell_metadata --cell-metadata-file ../tests/data/10k_cells_29k_genes.metadata.tsv
+
 # Ingest dense file
 $python ingest_pipeline.py ingest_expression --matrix-file ../tests/data/dense_matrix_19_genes_100k_cells.txt --matrix-file-type dense
```

Comment:
The prepended `$` has somewhat nagged me, because it requires extra care when copying and pasting.  So I'm glad you omitted it in your most recent example commands.  (Also, thank you for including example commands :).)

For consistency could you please remove `$` from the old examples, or add it to the new ones?

Reply:
Resolved in 5a5e6d0